rq_scheduler/utils.py

```diff
@@ -1,5 +1,6 @@
 import calendar
 import croniter
+import dateutil.tz
 
 import datetime
```

Comment:
useless import as you importing datetime class from datetime module literally next line

Reply:
It's used right here: https://github.com/rq/rq-scheduler/pull/248/files#diff-7ba69b7e087b1eaf1854726f6257bfafR27

Reply:
It can't be used anywhere cause use overriding this import importing datetime from datetime in next line..

Reply:
How does importing `datetime` override a `dateutil.tz` import?

Reply:
@Eugeny man.. I left the comment for 5th line..
`import datetime`
`from datetime import datetime, ..`

Reply:
@TsimafeiTsykunou gotcha. Feel free to post a PR.